src/notes/FluxNotesEditor.scss

```diff
@@ -0,0 +1,161 @@
+            padding-top:0;
+            font-size: .8rem;
+            margin: 0;
+        }
+    
+        p.note-name {
+            text-align: left;
+            padding-top:0;
+            font-size: .8rem;
+            margin: 0px 0px 0px 10px;
+            font-weight: bold;
+            
+            #edit-note-name-btn:hover {
+                cursor: pointer;
+            }
+            
+        }
+
+        .close-note-btn {
+            background-color: white !important;
+            padding: 0px !important;
+            margin-top: 0px !important;
+            text-transform: none !important;
+        }
+        
+        .close-note-btn:hover {
+            background: #E6E6E6 !important;
+        }
+
+        .note-description-divider {
+            margin: 20px 0px !important;
+            clear: both;
+        }
+    }
+    
+    .MyEditor-root {
+        font-family: 'Georgia', serif;
+
+        .toolbar-menu > * {
+            display: inline-block;
+        }
+
+        .toolbar-menu {
+            padding: 1px 0 17px 5px;
+            margin-bottom: 20px;
+            border-bottom: 1px solid #ddd;
+
+            .button {
+                color: #ccc;
+                cursor: pointer;
+            }
+
+            .button[data-active="true"] {
+                color: black;
+            }
+        }
+
+        p {
+            display: inline;
+            margin:0;
+        }
+
+        span {
+            display: inline;
+        }
+        
+    }
+}
+
+.editor-content {
+    /* 96px is the height of the Patient Control Panel, 20px is the height of the top margin of post encounter view
+    content and a little extra for potential scroll bars, 130px accounts for the editor header and editor toolbar 
+    plus margins, 76px is the height of the sign button, 20px is the margin on the sign button */
+    height: calc(100vh - 96px - 20px - 130px - 76px - 20px) !important;
+    overflow-y: auto;
+    overflow-x: hidden;
+    text-align: left;
+}
+
+.editor-content-modal {
+    /* 80vh for the modal height, 40px for modal padding, 40px for note-description */
+    max-height: calc(80vh - 40px - 40px);
+    overflow-y: auto;
+    overflow-x: hidden;
+    text-align: left;
+}
+
+// ******** StructuredFieldPlugin.jsx uses this
+.structured-field {
+    background-color: #d9ebf9;
+}
+
+.placeholder {
+    background-color: #dddddd;
+}
+
+// ******** suggestion-portal.js uses this
+.suggestion-portal { 
+    position: absolute;
+    box-sizing: border-box;
+    border-radius: 2px;
+    margin-top: 30px;
+    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+    box-shadow: rgba(0, 0, 0, 0.12) 0px 1px 6px, rgba(0, 0, 0, 0.12) 0px 1px 4px;
+    transition: all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;
+    background-color: rgb(255, 255, 255);
+    z-index: 1111;
+    max-height: 212px; /* 6 list elements at 32px each, plus 10px of margin on top and bottom */
+    overflow-y: auto;
+}
+
+.suggestion-portal ul {
```

Comment:
[JA1] Should we nest these styles like the #clinical-notes styles above it? Or is that part of the separate Jira you created?